packages/blockchain-wallet-v4-frontend/src/data/analytics/types.ts

```diff
@@ -1,14 +1,15 @@
 import * as AT from './actionTypes'
+import { AB_TESTS } from './model'
 import { MempoolFeeType } from 'data/components/types'
 import { RemoteDataType } from 'core/types'
 
 // types
-export type ABTestNameType = ''
+export type ABTestNameType = AB_TESTS
 
 export type ABTestCmdType = {
   command: MempoolFeeType
```

Comment:
`MempoolFeeType` seems wrong. I think that type can probably be deleted, just make a new one for your email test